Makefile

```diff
@@ -50,9 +57,13 @@ postsubmit-conformance:
 		--dry-run=false
 	bash development/kops/prow.sh
 
+.PHONY: upload
+upload:
+	release/s3_sync.sh $(RELEASE_BRANCH) $(RELEASE) $(ARTIFACT_BUCKET)
```

Comment:
If we are passing the ARTIFACT_BUCKET here anyways, let's not export it above then.

Reply:
I wrote a little test Makefile setup with a subdirectory and a shell script and exporting add passing these arguments seems to be unnecessary